sbin/prepareWorkspace.sh

```diff
@@ -37,14 +37,52 @@ ALSA_LIB_CHECKSUM=${ALSA_LIB_CHECKSUM:-5f2cd274b272cae0d0d111e8a9e363f0878332915
 ALSA_LIB_GPGKEYID=${ALSA_LIB_GPGKEYID:-A6E59C91}
 FREETYPE_FONT_SHARED_OBJECT_FILENAME="libfreetype.so*"
 
+
+copyFromDir() {
```

Comment:
The original clonnign script is using the `"." ` too, if that is what you are reffereing.  I was wondering too that there is no ful apth, but just `"." `  

I will heapily switch to full path instead of "." but allt he cloning methods were really exppecting other parts to `cd` for them. Can you confirm please?